### PR TITLE
AS-789 followup - do not post on bumps

### PR DESCRIPTION
**Pull request by ungwudik**

### CODE REVIEW

File: .github/workflows/pr-integration.yml [hunk 1]
```diff
@@ -96,8 +96,15 @@ jobs:
...
+    - name: Skip version bump merges
+      id: bump-check
+      uses: ./.github/actions/bump-skip
+      with:
+        event-name: ${{ github.event_name }}
+
     - name: "Notify WSM Slack"
-      if: always() && github.event_name == 'push' && github.ref == 'refs/heads/main'
+      # post to WSM Slack when a regular push (i.e. non-bumper push) is made to main branch
+      if: always() && github.event_name == 'push' && github.ref == 'refs/heads/main' && steps.bump-check.outputs.is-bump == 'no'
```
Comment (ddietterich) [inline .github/workflows/pr-integration.yml:107]: I do not think you need the `github.ref == 'refs/heads/main'` condition. Testing for `'push'` should be sufficient.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -172,7 +172,8 @@ jobs:
         fields: repo,job,workflow,commit,eventName,author,took
 
     - name: "Notify WSM Slack"
-      if: always() && github.event_name == 'push' && github.ref == 'refs/heads/main'
+      # post to WSM Slack when a regular push (i.e. non-bumper push) is made to main branch
+      if: always() && github.event_name == 'push' && github.ref == 'refs/heads/main' && steps.skiptest.outputs.is-bump == 'no'
```
Comment (ddietterich) [inline .github/workflows/test.yml:176]: ditto

**Review by ddietterich — APPROVED** [review]
Looks fine. Two comments.

File: .github/workflows/pr-integration.yml (hunk 1 above)
Comment (ungwudik) [reply .github/workflows/pr-integration.yml]: Oh yeah, that's true, since the `on push` trigger is only targeted for main branch.

**Review by zloery — APPROVED** [review]

**Pull request merged**